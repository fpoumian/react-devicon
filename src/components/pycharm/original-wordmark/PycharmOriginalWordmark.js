import React from 'react'
import PropTypes from 'prop-types'
import SVGDeviconInline from '../../_base/SVGDeviconInline'

import iconSVG from './PycharmOriginalWordmark.svg'

/** PycharmOriginalWordmark */
function PycharmOriginalWordmark({ width, height, className }) {
  return (
    <SVGDeviconInline
      className={'PycharmOriginalWordmark' + ' ' + className}
      iconSVG={iconSVG}
      width={width}
      height={height}
    />
  )
}

PycharmOriginalWordmark.propTypes = {
  className: PropTypes.string,
  width: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
  height: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
}

export default PycharmOriginalWordmark
