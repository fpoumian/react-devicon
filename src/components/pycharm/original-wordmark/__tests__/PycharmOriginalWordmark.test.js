import React from 'react'
import renderer from 'react-test-renderer'
import PycharmOriginalWordmark from '../index.js'

it('renders correctly', () => {
  const tree = renderer.create(<PycharmOriginalWordmark />).toJSON()
  expect(tree).toMatchSnapshot()
})
