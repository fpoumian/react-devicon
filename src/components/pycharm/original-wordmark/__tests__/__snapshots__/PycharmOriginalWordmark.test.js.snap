// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  className="SVGInline PycharmOriginalWordmark"
  dangerouslySetInnerHTML={
    Object {
      "__html": "<svg class=\\"SVGInline-devicon PycharmOriginalWordmark-devicon\\" id=\\"pycharm\\" xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 128 128\\"><defs><style>.cls-1{fill:#4d8548}.cls-2{fill:#fa930d}</style></defs><title>Artboard 12</title><g id=\\"original-wordmark\\"><path class=\\"cls-1\\" d=\\"M43.18 59.67a22 22 0 0 1 3.66-.29A6.07 6.07 0 0 1 51 60.61a4 4 0 0 1 1.29 3.09 4.36 4.36 0 0 1-1.14 3.16 6.13 6.13 0 0 1-4.52 1.62 6.52 6.52 0 0 1-1.54-.13v5.92h-1.91zm1.91 7.13a6.38 6.38 0 0 0 1.58.15c2.3 0 3.71-1.12 3.71-3.16S49 60.9 46.89 60.9a8 8 0 0 0-1.8.15zM55.57 63.66l2.32 6.27c.24.7.5 1.54.68 2.17.2-.64.42-1.45.68-2.22l2.11-6.23h2l-2.9 7.57c-1.38 3.64-2.33 5.51-3.64 6.65a5.23 5.23 0 0 1-2.37 1.25l-.48-1.62a5.11 5.11 0 0 0 1.69-.94 5.87 5.87 0 0 0 1.62-2.15 1.38 1.38 0 0 0 .15-.46 1.69 1.69 0 0 0-.13-.5l-3.93-9.78zM75.9 73.86a9.53 9.53 0 0 1-3.8.64c-5.11 0-7.74-3.18-7.74-7.39 0-5 3.6-7.85 8.07-7.85a8.72 8.72 0 0 1 3.64.66l-.68 2.65a7.2 7.2 0 0 0-2.8-.57c-2.65 0-4.72 1.6-4.72 4.89 0 3 1.75 4.82 4.74 4.82a8.35 8.35 0 0 0 2.79-.48zM77.92 58.71h3.33v6.12a4 4 0 0 1 1.32-1.1 3.74 3.74 0 0 1 1.73-.42c2.13 0 3.75 1.47 3.75 4.69v6.27h-3.29v-5.9C84.76 67 84.28 66 83 66a1.77 1.77 0 0 0-1.67 1.16 2.12 2.12 0 0 0-.11.75v6.36h-3.3V58.71zM96.86 74.28l-.2-1.07h-.06a3.86 3.86 0 0 1-3.07 1.32 3.29 3.29 0 0 1-3.47-3.29c0-2.79 2.5-4.12 6.3-4.1V67c0-.57-.31-1.38-2-1.38a6 6 0 0 0-3 .81l-.61-2.15a8.86 8.86 0 0 1 4.25-.97c3.58 0 4.72 2.11 4.72 4.63v3.73a16.55 16.55 0 0 0 .15 2.61zm-.39-5.07c-1.75 0-3.11.39-3.11 1.69a1.2 1.2 0 0 0 1.32 1.27 1.82 1.82 0 0 0 1.73-1.23 2.39 2.39 0 0 0 .07-.57v-1.16zM102.35 67.08c0-1.58 0-2.61-.09-3.53h2.87l.11 2h.09a3.15 3.15 0 0 1 2.9-2.22 3.43 3.43 0 0 1 .7 0v3.14a4.76 4.76 0 0 0-.9-.09 2.14 2.14 0 0 0-2.28 1.69 3.91 3.91 0 0 0-.07.75v5.44h-3.33v-7.2zM110.68 67c0-1.34 0-2.48-.09-3.42h2.81l.15 1.45h.07a3.72 3.72 0 0 1 3.25-1.69 3 3 0 0 1 2.94 1.82 4.77 4.77 0 0 1 1.38-1.29 3.86 3.86 0 0 1 2-.53c2.06 0 3.62 1.45 3.62 4.65v6.32h-3.25v-5.87c0-1.56-.5-2.46-1.58-2.46a1.63 1.63 0 0 0-1.54 1.16 2.79 2.79 0 0 0-.13.86v6.27h-3.25v-6c0-1.36-.48-2.28-1.53-2.28a1.7 1.7 0 0 0-1.56 1.21 2.22 2.22 0 0 0-.13.83v6.25h-3.25V67z\\"/><path class=\\"cls-2\\" d=\\"M24 57.63h-.1A5.83 5.83 0 0 0 18.22 62a5 5 0 0 1-9.9-.94 4 4 0 0 1 0-.55v-.11-.21a15.81 15.81 0 0 1 8.38-10.82A12.16 12.16 0 0 0 1.16 60l-.05.13v17.78h7.2v-5.75A12.16 12.16 0 0 0 25 57.63h-1z\\"/><path class=\\"cls-1\\" d=\\"M17 61.76a7 7 0 0 1 6.8-5.32h3.44a3.73 3.73 0 0 0 1.55-.44h-.36a4.92 4.92 0 0 1-3.73-3.33 4.89 4.89 0 0 0 5.9 1.5 3.76 3.76 0 0 0-3.44-5.26h-3.34A14.55 14.55 0 0 0 9.61 60.32a3.71 3.71 0 0 0-.1.73 3.77 3.77 0 0 0 7.48.71zm9-10.71a1.17 1.17 0 1 1 .92 1.37 1.17 1.17 0 0 1-.92-1.37zm4.93 23.15a3.77 3.77 0 0 1-3.77 3.8h-3.34a14.47 14.47 0 0 1-9.52-3.56 13.35 13.35 0 0 0 8.6-4 7.26 7.26 0 0 0 .91.06h3.34a3.77 3.77 0 0 1 3.78 3.71z\\"/></g></svg>",
    }
  }
/>
`;
